frontend/src/pages/Homepage.jsx

```
@@ -3,22 +3,28 @@ import Navbar from '../Components/Navbar'
+            Increase your<br />
+            Result<br />
+            with us!
+          </h1>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do 
              eiusmod tempor incididunt ut labom et dolore magna aliqua. Ut 
              enim ad minim veniam</p>
-          <button className="explore-btn">
+          <button 
+            className="explore-btn"
+            onClick={() => navigate('/contributions')}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)